starter-code/pizza.js

```diff
@@ -1 +1,83 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+
+  $('.price strong').text('$13');
+  $('.price li:nth-last-child(-n+2)').toggle();
+  $('.btn-sauce').toggleClass('active');
+  $('.btn-crust').toggleClass('active');
+  togglePepperonni();
+  toggleMushroom();
+  toggleGreen();
+  toggleSauce();
+  toggleGluten();
+
+});
+
+function updateTotalPrice() {
+  var tmp = 10;
+  if ($('.btn-pepperonni').hasClass('active')) {
+    tmp += 1;
+  }
+  if ($('.btn-mushrooms').hasClass('active')) {
+    tmp += 1;
+  }
+  if ($('.btn-green-peppers').hasClass('active')) {
+    tmp += 1;
+  }
+  if ($('.btn-sauce').hasClass('active')) {
+    tmp += 3;
+  }
+  if ($('.btn-crust').hasClass('active')) {
+    tmp += 5;
+  }
+  totalPriceText = '$' + tmp;
+  $('.price strong').text(totalPriceText);
+  console.log(tmp);
+
+}
+
+
```

Comment:
the next functions could be transformed in a single function that receives params in order to handle the differences between the current functions